scripts/export_large_tsv/export_large_tsv.py

```diff
@@ -5,8 +5,10 @@
 import argparse
 import math
 
+default_page_size = 1000
```

Comment:
this is small but I've seen these "global" variables be in all caps to distinguish them clearly from local variables